app/src/main/java/com/example/demouniverload/search/presenter/MethodsOfYandexParser/YandexRequestsSender.java

```diff
@@ -0,0 +1,22 @@
+package com.example.demouniverload.search.presenter.MethodsOfYandexParser;
+
+import com.example.demouniverload.search.presenter.ParsersInterface;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+import java.io.IOException;
+
+public class YandexRequestsSender implements ParsersInterface.YandexMethodsInterface.IYandexRequestSender {
+    @Override
+    public Document requestToSite(String message) {//Request to site
```

Comment:
```suggestion
    public Document requestToSite(String message) {
```
This comment is pretty useless. It repeats the method's name.